tests/recipes/test_reportlab.py

```diff
@@ -0,0 +1,54 @@
+import os
+import unittest
+from mock import patch
+from pythonforandroid.archs import ArchARMv7_a
+from pythonforandroid.build import Context
+from pythonforandroid.graph import get_recipe_order_and_bootstrap
+from pythonforandroid.recipe import Recipe
+from pythonforandroid.util import ensure_dir
+
+
+class TestReportLabRecipe(unittest.TestCase):
+
+    def setUp(self):
```

Comment:
For now the `setUp()` looks a bit heavy, but as recipe unit tests grow we will easily move common setup things to common class for other recipe tests to use.
So the idea is not to have it perfect for the first iteration :smile: